posthog/utils.py

```diff
@@ -388,9 +389,26 @@ def render_template(template_name: str, request: HttpRequest, context: Dict = {}
 
     if posthog_distinct_id:
         groups = {}
-        if request.user and request.user.is_authenticated and request.user.organization:
-            groups = {"organization": str(request.user.organization.id)}
-        feature_flags = posthoganalytics.get_all_flags(posthog_distinct_id, only_evaluate_locally=True, groups=groups)
+        group_properties = {}
+        person_properties = {}
+        if request.user and request.user.is_authenticated:
+            user = cast("User", request.user)
+            person_properties["email"] = user.email
+            person_properties["joined_at"] = user.date_joined.isoformat()
```

Comment:
specially date time filters, we do parse these into date times, so should work, but please do test it works locally by creating a flag based on this property

Reply:
Works!